cgmanifest.json

```diff
@@ -28187,8 +28187,8 @@
         "type": "other",
         "other": {
           "name": "systemd",
-          "version": "250.3",
-          "downloadUrl": "https://github.com/systemd/systemd-stable/archive/v250.3.tar.gz"
+          "version": "254",
+          "downloadUrl": "https://github.com/systemd/systemd/archive/v254.tar.gz"
```

Comment:
When we get to AzLinux 3.0 release, is the thinking to stay with systemd upstream releases or go back to the systemd-stable releases?

Reply:
in m2, we weren't actually following the systemd stable releases (the latest stable-250 is 250.13), so if we continue that it doesn't really matter much if we pull from stable or non-stable.

But yes, I do think it would be a good idea for azl3 to actually follow systemd-stable releases; I'll change this to switch over to 254.6 and we can move over to stable-255 once that's out

Reply:
well, unfortunately the systemd-254.6.tar.gz in our 'blobstore' was added with the wrong upstream URL (which doesn't work), so there is no way for me to get the PR check in this to work; if the spec file has the blobstore-registered URL then the PR check can't download the tarball (because the URL is wrong), and if the spec file has the correct URL then it doesn't match the registered URL and the PR check fails.

Hopefully someone can delete the broken blobstore tarball and i'll re-publish with the right url

Reply:
ok working now, not sure if it was my mistake or what, but i used 254.5 instead and it's working ok